packages/klighd-core/src/views-rendering.tsx

```diff
@@ -187,6 +186,7 @@ export function renderRectangularShape(rendering: KContainerRendering, parent: S
             } else {
                 // Fallthrough to KEllipse case.
             }
+            break
```

Comment:
As seen by the comment above, this fall-through without a break was put there purposefully, so this should stay the same

Reply:
Well, that was properly auto fixed. In general, I think eslint was right. Having a fall through in a case with such enormous code block seems pretty likely to lead to future bugs.